Documentation/project-docs/developer-guide.md

```diff
@@ -87,50 +88,33 @@ The native component should be buildable on any system.
 
 - Building in debug mode for platform x64
 ```
-build-native -debug -buildArch=x64
+./src/Native/build-native debug x64
 ```
 
-- The following example shows the argument `--`. Everything that is after it is not going to be processed, and will be passed as-is.
+- The following example shows how you would do an arm cross-compile build.
 ```
-build-native -debug -buildArch=arm -- cross verbose
+./src/Native/build-native debug arm cross verbose
 ```
 
 For more information about extra parameters take a look at the scripts `build-native` under src/Native.
 
-### Build Managed
-Since the managed build uses the .NET Core CLI (which the build will download), managed components can only be built on a subset of distros.
-There are some additional prerequisites from the CLI which need to be installed. Both libicu and libunwind are used by CoreCLR to execute managed code, so they must be installed. Since CoreFX does not actually link against these packages, runtime versions are sufficient. We also require curl to be present, which we use to download the .NET Core CLI.
-
-**Examples**
-
-- Building in debug mode for platform x64
-```
-build-managed -debug -buildArch=x64
-```
-
-- Building in debug mode for platform x64 targeting OS Linux
-```
-build-managed -debug -buildArch=x64 -os=Linux
-```
-
 ### Build And Run Tests
-To build the tests and run them you can call the build-test script. The same parameters you pass to build should also be passed to build-tests script to ensure you are building and running the tests on the same configuration you have build the product on. However to run tests on the same machine you need to ensure the machine supports the configuration you are building.
+To build the tests and run them you can call the build script passing -tests option. The same parameters you pass to build should also be passed to ensure you are building and running the tests on the same configuration you have build the product on. However to run tests on the same machine you need to ensure the machine supports the configuration you are building.
 
 **Examples**
-- The following shows how to build tests but not run them
+- The following shows how to build only the tests but not run them
 ```
-build-tests -skiptests
+build -test -skiptests
```

Comment:
-includetests

Reply:
These steps were only building tests before so the `-test` option is the correct option `-includetests` will build both product and tests.